db/migrate/20150522121048_add_command_type_to_stages.rb

```diff
@@ -0,0 +1,7 @@
+class AddCommandTypeToStages < ActiveRecord::Migration
+  def change
+    change_table :stages do |t|
+      t.string :command_type
```

Comment:
I was a bit leery of doing that as at the time I was changing the class name a lot with indecisiveness, so not giving a default allows the default to be in code and thus refactorings become easier in the future and you then only have to worry about the non-default rows.